### ISSUE TITLE
Missing Set as wide page option on image Haptic Touch

### ISSUE THREAD

**Issue by EzWiz21** [Aidoku/Aidoku#194]
Setting an image as a wide page was an option available in previous builds (I don’t remember which one specifically) but it’s now missing.

**Comment by Skittyblock** [comment]
yeah it was a feature for a bit, but I ended up having to rewrite the entire system and I never added that back in

**Comment by omeryagmurlu** [comment]
Any plans to add this back? This is the only pain point I have with the app coming from paperback

**Comment by Skittyblock** [comment]
added back with #697
